src/v/cloud_roles/apply_abs_oauth_credentials.h

```diff
@@ -11,7 +11,7 @@
 #pragma once
 
 #include "cloud_roles/apply_credentials.h"
-#include "cloud_roles/signature.h"
+#include "signature.h"
 
```

Comment:
Ah now i get it. But does it have some practical purposes, or is it the base for some future transformations?

Reply:
We're going to remove the `src/v` include path in the future to prevent people from create dependency cycles via header includes.